lxd/storage/drivers/driver_lvm_utils.go

```diff
@@ -60,7 +60,7 @@ func (d *lvm) openLoopFile(source string) (*os.File, error) {
 		defer unlock()
 
 		// Try to prepare new loop device.
-		loopF, err := PrepareLoopDev(source, 0)
+		loopF, err := PrepareLoopDev(source, LoFlagsDirectIO)
```

Comment:
Where is that argument defined? This should fail to compile given it's the first commit in the series?

Reply:
We require that all commits build without an error/warning not just the whole series/branch. To make sure that this works you can do:

```
git rebase origin/master -x "make -j4"
```

Reply:
(from your branch that is)

Reply:
I have it defined in `utils-cgo.go` on line 291. Should I move it into the driver files?

Reply:
I restructured the commits. They should all build without errors now.